test/plugins/dot-net.test.js

```diff
@@ -0,0 +1,18 @@
+import assert from 'assert';
+import dotNet from '../../lib/plugins/dot-net';
+
+describe('dotNet', () => {
+  it('resolves EntityFramework to https://www.nuget.org/packages/EntityFramework', () => {
+    assert.deepEqual(
+      dotNet.resolve({ target: 'EntityFramework' }),
+      ['https://www.nuget.org/packages/EntityFramework'],
+    );
+  });
+
+  it('resolves Stanford.NLP.CoreNLP to https://www.nuget.org/packages/Stanford.NLP.CoreNLP', () => {
+    assert.deepEqual(
+      dotNet.resolve({ target: 'Stanford.NLP.CoreNLP' }),
+      ['https://www.nuget.org/packages/Stanford.NLP.CoreNLP'],
+    );
+  });
+});
```

Comment:
Could you also add some fixtures here in order to make sure that the `NET_PACKAGE` regex matches the package name correctly? https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/test.js#L6